src/parser/ad_parser.js

```diff
@@ -143,6 +167,28 @@ function parseAdElement(adTypeElement, emit) {
  */
 function parseWrapper(wrapperElement, emit) {
   const ad = parseAdElement(wrapperElement, emit);
+
+  const followAdditionalWrappersValue = wrapperElement.getAttribute(
+    'followAdditionalWrappers'
+  );
+  const allowMultipleAdsValue = wrapperElement.getAttribute('allowMultipleAds');
+  const fallbackOnNoAdValue = wrapperElement.getAttribute('fallbackOnNoAd');
+
+  ad.followAdditionalWrappers =
+    followAdditionalWrappersValue !== ''
```

Comment:
`getAttribute` can return `''` or `null`: https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute, so we should check for both here and below

Reply:
Oops, indeed.